tools/dds/dds-server/CMakeLists.txt

```diff
@@ -2,24 +2,27 @@
 # Copyright(c) 2022 Intel Corporation. All Rights Reserved.
 project(rs-dds-server)
 
-set(SRC rs-dds-server.cpp
+set(TOOL_FILES rs-dds-server.cpp
         dds-server.cpp
-        ${REPO_ROOT}/src/dds/msg/devicesPubSubTypes.cpp # TODO Consider moving it to a common folder
-        ${REPO_ROOT}/src/dds/msg/devices.cpp)
+        dds-server.h)
         
-set(INC dds-server.h
-        ${REPO_ROOT}/src/dds/msg/devicesPubSubTypes.h
-        ${REPO_ROOT}/src/dds/msg/devices.h)
+set(TOPICS_FILES 
```

Comment:
Why not use `include/librealsense2/dds/CMakeLists.txt`?

Reply:
Changed